helpers/dishes-model.js

```diff
@@ -0,0 +1,47 @@
+const db = require('../data/dbConfig');
+
+module.exports = {
+    find,
+    findById,
+    add,
+    update,
+    remove,
+};
+
+function find() {
+    return db('dishes');
+}
+
+function findById(id) {
+    return db('dishes')
+      .where({ id })
+      .first();
+}
+
+function add(dish) {
+    return db('dishes')
+      .insert(dish, 'id')
+      .then(([id]) => {
+        return findById(id);
```

Comment:
findById is a database call which is asynchronous. There should be another .then to handle the promise.